frontend/src/components/Home/Contributors/card.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Check if we can use material-ui card

Reply:
material-ui card required css to be overridden and code size is almost same, so I think its better to use custom card for now.